lib/hammer_cli_katello/id_resolver.rb

```diff
@@ -137,7 +139,7 @@ def content_view_version_id(options)
         results_in_from_environment
           .sort { |a, b| a['version'].to_f <=> b['version'].to_f }.last['id']
       else
-        pick_result(results, @api.resource(:content_view_versions))['id']
+        nil
```

Comment:
Lets add this so Rubocop is happy:
`# rubocop:disable Style/EmptyElse`

Reply:
Thanks, fixed.